source/layers/validation/ze_validation_layer.cpp

```diff
@@ -15,6 +15,53 @@ namespace validation_layer
 {
     context_t& context = context_t::getInstance();
 
+    void context_t::initLogger() try {
+        std::string loggingLevel = "error";
+        std::string loggingFormat = "ZE ---> %v";
```

Comment:
Should we use the same default format as ze loader here? ([date] [logger_name] [level] message...)